src/renderers/webgl/WebGLShadowMap.js

```diff
@@ -70,10 +71,12 @@ function WebGLShadowMap( _renderer, _lights, _objects, capabilities ) {
 
 		var useMorphing = ( i & _MorphingFlag ) !== 0;
 		var useSkinning = ( i & _SkinningFlag ) !== 0;
+		var useInstancing = ( i * _InstancingFlag ) !==0;
```

Comment:
it should, but, like i said, i dont think there is a need to do any of this any more, a material could be decorated with a define, and there is no need to change anything in order for this to work

Should also be `useInstance` same length as the other two :D